Objects/codeobject.c

```diff
@@ -2268,3 +2268,57 @@ _PyStaticCode_InternStrings(PyCodeObject *co)
     }
     return 0;
 }
+
+PyCodeObject *
+_Py_MakeTrampoline(const char *code, int codelen, int stacksize, const char *cname)
+{
+    PyObject *name = NULL;
+    PyObject *co_code = NULL;
+    PyObject *lines = NULL;
+    PyCodeObject *codeobj = NULL;
+
+    name = _PyUnicode_FromASCII(cname, strlen(cname));
+    if (name == NULL) {
+        goto cleanup;
+    }
+    co_code = PyBytes_FromStringAndSize(code, codelen);
+    if (co_code == NULL) {
+        goto cleanup;
+    }
+    const char loc[2] = { 0x80 | (PY_CODE_LOCATION_INFO_NO_COLUMNS << 3) | 3, 0 };
```

Comment:
For example, this linetable makes some pretty serious assumptions about the size `code`/`codelen`.

Also, I think it's wrong in this case (`INTERPRETER_TRAMPOLINE_CODE` has three code units, but this table describes locations for four code units).

Reply:
(Also, another place we should probably use `uint8_t`.)